src/main.py

```diff
@@ -591,7 +601,7 @@ def run(settings, components, variables, pyomoChk=None):
   vprint(v, 0, m, 'Starting CashFlow Run ...')
   # check mapping of drivers and determine order in which they should be evaluated
   vprint(v, 0, m, '... Checking if all drivers present ...')
-  ordered = checkDrivers(settings, components, variables, v=v)
+  ordered = checkDrivers(settings, components, variables, v=v, pyomoDriv=pyomoChk)
```

Comment:
I noticed this flag changes names a few times. Would it be useful to make it consistent so that maintaining this feature is easier in the future? Or do the different flags do different things that I'm not seeing?

Reply:
I have it changing names just in case if someone changes the code in the future, the location of the error related to pyomo flags would be obvious.